decode/src/monitors.rs

```diff
@@ -277,7 +303,13 @@ impl ParseFromModule for Monitors {
     fn reads(id: Identifier) -> Result<Vec<MemoryRead>, Error> {
         match id {
             Identifier::QsfpPlusSff8636 | Identifier::Qsfp28 => {
-                // First, we'll read support for monitoring values, and the
+                // First, read the revision compliance byte. This will tell us
+                // how to handle the fact that we might be missing advertisement
+                // bits for the temperature and voltage.
+                let page = sff8636::Page::Lower;
+                let status = MemoryRead::new(page, 1, 1).unwrap();
```

Comment:
small nit to include an SFF-8636 table reference since we do that for all the other reads too. Looks like Table 6-3.

Reply:
Done in 8f46e98